local-stack/README.md

```diff
@@ -26,53 +26,111 @@ Components installed :
 - Terraform
 - [Yq](https://mikefarah.gitbook.io/yq) v4.x
 - Conduktor License in `LICENSE` environment variable set.
+- Kafka CLI commands (e.g. `brew install kafka`)
 
-## 1. Create cluster with base components
+## Create cluster with base components
 
 To create and start the local environment, run the following commands :
 ```bash
 make start-local-stack
 ```
 It will create a K3d cluster named k3d-conduktor-platform-p75 and install base components (Ingress controller, Cert-manager, Monitoring stack, Postgresql databases, Kafka cluster, Vault).
-Kubectl context should be created. To use it run `kubectl config use-context k3d-conduktor-platform-p75`.
+Kubectl context should be created. To use it, run 
+
+```
+kubectl config use-context k3d-conduktor-platform-p75
+```
+
+## Deploy Conduktor platform
 
-## 2. Deploy Conduktor platform
 Then, to install Conduktor Console and Gateway, run the following commands :
 ```bash
 make install-conduktor-platform
 ```
 It will deploy [`console-secrets`](local-stack/console-secrets.yaml) and [`gateway-secrets`](local-stack/gateway-secrets.yaml) into `conduktor` namespace and
 then install both Conduktor Console and Gateway latest helm charts using [`console-values`](local-stack/console-values.yaml) and [`gateway-values`](local-stack/console-values.yaml) files.
 
-## 3. Access Conduktor platform
-Add the following line to your `/etc/hosts` file :
+
+## Provision Conduktor platform using terraform
+
+To provision the Conduktor platform using terraform, run the following commands :
+```bash
+make init-conduktor-platform
+```
+Provisioning create resources inside Conduktor Console and Gateway.
+
+## Access Conduktor platform
+Add the following line to your `/etc/hosts` file in order to resolve hostnames:
```

Comment:
you need to update /etc/hosts before provisioning or terraform will not be able to reach APIs